### PR TITLE
update qtype mapping

### PR DESCRIPTION
**Pull request by leoparente**

### CODE REVIEW

File: libs/visor_dns/dns.h [hunk 1]
```diff
@@ -108,14 +110,18 @@ static std::unordered_map<uint16_t, std::string> QTypeNames({
     {252, "AXFR"},
     {253, "MAILB"},
     {254, "MAILA"},
+    {254, "*"},
```
Comment (rboucher-me) [inline libs/visor_dns/dns.h]: shouldn't this be 255?

File: libs/visor_dns/dns.h [hunk 2]
```diff
@@ -196,11 +203,15 @@ static std::unordered_map<std::string, uint16_t> QTypeNumbers({
     {"AXFR", 252},
     {"MAILB", 253},
     {"MAILA", 554},
+    {"*", 254},
```
Comment (rboucher-me) [inline libs/visor_dns/dns.h]: 255?

File: libs/visor_dns/dns.h [hunk 3]
```diff
@@ -28,6 +28,7 @@ enum RCode {
 };
 
 static std::unordered_map<uint16_t, std::string> QTypeNames({
+    {0, "Reserved"},
```
Comment (rboucher-me) [inline libs/visor_dns/dns.h]: Would it make sense to label this "Reserved (0)" to help differentiate from the other reserved qtypes?

File: libs/visor_dns/dns.h [hunk 4]
```diff
@@ -108,14 +110,18 @@ static std::unordered_map<uint16_t, std::string> QTypeNames({
...
     {254, "MAILA"},
+    {254, "*"},
     {256, "URI"},
     {257, "CAA"},
     {258, "AVC"},
     {259, "DOA"},
-    {260, "AMTRELAY"},
+    {32768, "TA"},
+    {32769, "DLV"},
+    {65535, "Reserved"},
```
Comment (rboucher-me) [inline libs/visor_dns/dns.h]: "Reserved (65535)"?

**Review by rboucher-me — COMMENTED** [review]
See comments

File: libs/visor_dns/dns.h [hunk 5]
```diff
@@ -196,11 +203,15 @@ static std::unordered_map<std::string, uint16_t> QTypeNumbers({
     {"AXFR", 252},
     {"MAILB", 253},
     {"MAILA", 554},
```
Comment (rboucher-me) [inline libs/visor_dns/dns.h]: 554? shouldn't it be "254"?

File: libs/visor_dns/dns.h (hunk 1 above)
Comment (leoparente) [reply libs/visor_dns/dns.h]: yes, fixed

File: libs/visor_dns/dns.h (hunk 2 above)
Comment (leoparente) [reply libs/visor_dns/dns.h]: yes, fixed

File: libs/visor_dns/dns.h (hunk 4 above)
Comment (leoparente) [reply libs/visor_dns/dns.h]: yes, fixed

File: libs/visor_dns/dns.h (hunk 5 above)
Comment (leoparente) [reply libs/visor_dns/dns.h]: yes, fixed

File: libs/visor_dns/dns.h (hunk 3 above)
Comment (leoparente) [reply libs/visor_dns/dns.h]: yes, fixed

**Review by weyrick — APPROVED** [review]

**Pull request merged**